app.js

```diff
@@ -49,18 +48,45 @@ function Runner(
                 return skuPack.start(configuration.get('skuPackRoot', './skupack.d'));
             })
             .then(function() {
-                return app.listen();
+                // TODO: The following endpoint definition is for DEBUG using
+                // It will fianlly been moved to monorail.json config file
+                // var endpoints = configuration.get('httpEndpoints', [{port:8080}]);
```

Comment:
Or, at the very least, use `configuration.get` in the endpoints definition, for example:

``` 
{
    "address": configuration.get("apiServerAddress", "172.31.128.1",
    "port": configuration.get("apiServerPort", "9080",
    // etc.
}
```